docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+---
+services:
+  # run postgres for proxy service to store observability metrics
+  postgres:
+    image: postgres:15
+    env_file: .env
+    ports:
+      - "${POSTGRES_HOST_PORT}:${POSTGRES_CONTAINER_PORT}"
+  # run redis for proxy service to cache responses
+  redis:
+    image: 'bitnami/redis:latest'
+    env_file: .env
+    ports:
+      - "${REDIS_HOST_PORT}:${REDIS_CONTAINER_PORT}"
+  # run single validator kava node to provide a local kava network for development and testing of the proxy service
+  kava:
+    image: kava/kava:latest
+    entrypoint: /docker/kava/kava-entrypoint.sh
+    env_file: .env
+    volumes:
+      - ./docker/kava:/docker/kava
+    ports:
+      - "${KAVA_HOST_COSMOS_RPC_PORT}:${KAVA_CONTAINER_COSMOS_RPC_PORT}"
+      - "${KAVA_HOST_EVM_RPC_PORT}:${KAVA_CONTAINER_EVM_RPC_PORT}"
+  # run proxy service to observe, route, and scale requests to kava api endpoints
+  proxy:
+    build:
+      dockerfile: LocalDockerfile
+    env_file: .env
```

Comment:
I believe `.env` is automatically loaded by docker-compose

Reply:
correct, but I'm an adherent to the zen of python and go proverbs of "explicit is better than implicit", that way if some none docker compose wizard has to work with this file they have one less thing to be confused about (hopefully)